### PR TITLE
adding new parameters for VAS 2012

### PR DESCRIPTION
**Pull request by dipyayan**
creating this PR for adding new parameters for VAS 2012.

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/knxQUlZn](https://trello.com/c/knxQUlZn)

File: content/api/shipping-guide_2/_index.md [hunk 1]
```diff
@@ -129,6 +129,12 @@ documentation:
                         <ns:AdditionalServices>
                             <ns:AdditionalService>
                                 <ns:Id>EVARSLING</ns:Id>
+                                <ns:LeadTimeFromCustomerInMinutes>30</ns:LeadTimeFromCustomerInMinutes>
+                                <ns:SortingAreas>
+                                    <ns:String>1000</ns:String>
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:134]: values here can be :

image

can we change the values?

File: content/api/shipping-guide_2/_index.md [hunk 2]
```diff
@@ -219,7 +225,9 @@ documentation:
             ],
             "additionalServices": [
               {
-                "id": "EVARSLING"
+                "id": "EVARSLING",
+                "leadTimeFromCustomerInMinutes": 30,
+                "sortingAreas": ["1000","2000","3000","4000"]
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:230]: same here

File: content/api/booking/examples/vas/2012.xml [hunk 3]
```diff
@@ -1,5 +1,12 @@
 <additionalServices>
   <additionalService>
     <id>2012</id>
+    <leadTimeFromCustomerInMinutes>30</leadTimeFromCustomerInMinutes>
+    <sortingAreas>
+      <String>1000</String>>
+      <String>2000</String>>
```
Comment (AnshikaKoul) [inline content/api/booking/examples/vas/2012.xml:7]: remove the examples from booking?

**Comment by AnshikaKoul** [comment]
can we add something about the parameters in the documentation?

File: content/api/booking/examples/vas/2012.xml (hunk 3 above)
Comment (dipyayan) [reply content/api/booking/examples/vas/2012.xml:7]: done.

File: content/api/shipping-guide_2/_index.md (hunk 2 above)
Comment (dipyayan) [reply content/api/shipping-guide_2/_index.md:230]: done.

File: content/api/shipping-guide_2/_index.md (hunk 1 above)
Comment (dipyayan) [reply content/api/shipping-guide_2/_index.md:134]: done.

File: content/api/shipping-guide_2/_index.md [hunk 4]
```diff
@@ -691,6 +700,34 @@ documentation:
       "customerNumber": "2001001000",
       ```
 
+  - title:
+    2012 VAS for Same day delivery for products 5600 and 1706
+    content: |
+    If you are sending a request for same day delivery using 2012 VAS, then two additional parameters need to be added in the request alongside the VAS.They are **leadTimeFromCustomerInMinutes** which will be the lead time in minutes and **sortingAreas** which will represent the area codes in sorted order.
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:706]: @dipyayan can we add text thea shared with us here?

Screenshot 2022-10-03 at 13 29 09

Comment (dipyayan) [reply content/api/shipping-guide_2/_index.md]: done.

File: content/api/shipping-guide_2/_index.md [hunk 5]
```diff
@@ -691,6 +700,32 @@ documentation:
...
+      Sorting area for Same day delivery (2012)
+    content: |
+      The Same day delivery VAS (2012) allows shipments to be delivered same- or next-day, depending on when the shipment is booked. By default, SG API returns the full service coverage area for Same day delivery. The "sortingAreas" parameter allows the web shop to reduce the delivery area to the area they currently choose to provide the service."routeInformation" contains information about the sorting area for a specific parcel. Intended use is to aid pre-sorting of same day parcels.
+      * SOAP: 
+      ```xml
+      <ns:AdditionalService>
+        <ns:Id>EVARSLING</ns:Id>
+        <ns:LeadTimeFromCustomerInMinutes>30</ns:LeadTimeFromCustomerInMinutes>
+        <ns:SortingAreas>
+            <ns:String>100</ns:String>
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:713]: is this string? or area? can you check in the code?

Screenshot 2022-10-03 at 13 50 35

File: content/api/shipping-guide_2/_index.md [hunk 6]
```diff
@@ -129,6 +129,13 @@ documentation:
                         <ns:AdditionalServices>
                             <ns:AdditionalService>
                                 <ns:Id>EVARSLING</ns:Id>
+                                <ns:LeadTimeFromCustomerInMinutes>30</ns:LeadTimeFromCustomerInMinutes>
+                                <ns:SortingAreas>
+                                    <ns:String>100</ns:String>
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:134]: string? 🤔 

is this string? or area? can you check in the code?

Screenshot 2022-10-03 at 13 50 35

File: content/api/shipping-guide_2/_index.md [hunk 7]
```diff
@@ -219,7 +226,9 @@ documentation:
             ],
             "additionalServices": [
               {
-                "id": "EVARSLING"
+                "id": "EVARSLING",
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:229]: this isn't evarsling?

File: content/api/shipping-guide_2/_index.md [hunk 8]
```diff
@@ -129,6 +129,13 @@ documentation:
                         <ns:AdditionalServices>
                             <ns:AdditionalService>
                                 <ns:Id>EVARSLING</ns:Id>
```
Comment (AnshikaKoul) [inline content/api/shipping-guide_2/_index.md:131]: this isn't evarsling?

**Pull request merged**